artifactory/cli.go

```diff
@@ -1235,6 +1263,19 @@ func validateServerId(serverId string) {
 	}
 }
 
+func validateGoNativeCommand(args []string) string {
```

Comment:
I suggest adding a description comment for this function.
I think that it'll be better if this function actually returns an error, with the invalid flag it finds. It may make the function (and also its test) easier to understand.

Reply:
Done